crates/holochain/src/core/ribosome/host_fn/accept_countersigning_preflight_request.rs

```diff
@@ -29,75 +28,20 @@ pub fn accept_countersigning_preflight_request<'a>(
             if let Err(e) = input.check_integrity() {
                 return Ok(PreflightRequestAcceptance::Invalid(e.to_string()));
             }
-            let author = super::agent_info::agent_info(_ribosome, call_context.clone(), ())?
-                .agent_latest_pubkey;
             tokio_helper::block_forever_on(async move {
-                if (holochain_zome_types::prelude::Timestamp::now() + SESSION_TIME_FUTURE_MAX)
+                if (Timestamp::now() + SESSION_TIME_FUTURE_MAX)
                     .unwrap_or(Timestamp::MAX)
```

Comment:
Pretty unlikely but Rust makes you handle the error so that if somebody runs this in a few thousand years then this won't be how it fails... because that'll happen :P